app/src/main/java/com/android/unio/ui/settings/Settings.kt

```diff
@@ -169,6 +197,21 @@ fun SettingsContainer() {
                       Manifest.permission.ACCESS_COARSE_LOCATION))
             }
           })
+      preference(
```

Comment:
I personally believe a toast to indicate that a link  has been sent to reset the password is a better idea than the added text, and goes better with the rest of the app. But this is personal preference and your implementation works fine.